scripts/lc-builds/blueos_spectrum_nvcc_clang.sh

```diff
@@ -61,8 +61,8 @@ echo "***********************************************************************"
 echo
 echo "cd into directory build_${BUILD_SUFFIX} and run make to build RAJA Perf Suite"
 echo
-echo "  Please note that you have to disable CUDA GPU hooks when you run"
-echo "  the RAJA Perf Suite; for example,"
+echo "  Please note that you have to run with mpi and disable CUDA GPU hooks"
```

Comment:
it looks like you removed the lrun option for disabling the CUDA hooks.

Reply:
I should remove that from the description, its only necessary when running without mpi.